### PR TITLE
Resolved issue #263

### PR DESCRIPTION
**Pull request by hamorphis**

### CODE REVIEW

**Comment by hamorphis** [comment]
The new property is called Page.navigationBarHidden.

File: ui/page/page-common.ts [hunk 1]
```diff
@@ -36,9 +54,24 @@ export class Page extends contentView.ContentView implements dts.Page, view.AddA
 
     public onLoaded() {
         this._applyCss();
+        this._updateNavigationBar(this.navigationBarHidden);
         super.onLoaded();
     }
 
+    get navigationBarHidden(): boolean {
+        console.log("page-common get navigationBarHidden: " + this._getValue(Page.navigationBarHiddenProperty));
```
Comment (vakrilov) [inline ui/page/page-common.ts]: Remove console.log()-s

**Comment by vakrilov** [comment]
:+1:

**Comment by nsndeck** [comment]
:+1:

**Pull request merged**